test/units.test.js

```diff
@@ -3,6 +3,19 @@ import AddToDoList from '../src/addRemoveTask';
 const todoList = new AddToDoList();
```

Comment:
- [ ] Its a requirement that you use the mock storage object you have created in Part 1 of this project to mock the localStorage updates.
- [ ] Also mock the HTML elements to test DOM manipulation functions.